src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -432,18 +439,34 @@ public double getLinearSpeed() {
         @Override
         public void addVisionMeasurement(Pose2d pose, double timestampSeconds,
                         Matrix<N3, N1> visionMeasurementStdDevs) {
+                SmartDashboard.putNumber("addVisionMeasurement/timeStampSeconds", timestampSeconds);
```

Comment:
I think we can get rid of this. I think this was put in there because there was a thought that timestamp was causing problems. So we needed to see the value